scripts/translationScripts/utils.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require("path")
+
+const getAllFiles = function(dirPath, ext, arrayOfFiles = []) {
+    const files = fs.readdirSync(dirPath)
+
+    files.forEach(file => {
+        if (fs.statSync(dirPath + "/" + file).isDirectory()) {
```

Comment:
It's better to consisently use `path.join` instead of `+`'ing strings. `path.join` also normalizes the resulting path, which is nice.

Reply:
Oops, my bad, I just copy pasted and didn't double check the code